lang/en/docs/tutorials/materials/non-periodic.md

```diff
@@ -0,0 +1,102 @@
+# Create Non-Periodic Structures
```

Comment:
I think we don't need a tutorial on how to create a non-periodic structure. Adding content into the corresponding materials actions pages would be sufficient.